### ISSUE TITLE
Convert slick slider to swiper.js and other improvements

### ISSUE THREAD

**Issue by waysagency** [CrocoBlock/suggestions#5573]
Slick slider has become one of the worst sliders in developing world... Consider removing Slick sliders and converting to Swiper.js. Also, it's weird that you use swiper.js in woobuilder, but not in jetengine...

I would also like to see some more options like:
- Custom arrow container
- Slides to show with decimal numbers
- Free mode
- scrollbar

Or just give the possibility to write some json for the advanced users.

In any case, it is really a shame to see that all kinds of new features are being introduced, instead of the current ones being improved. There are so many things that just don't work (right) lately. In addition, I would also like to see more advanced documentation rather than just standard solutions. A lot should be possible, but little is possible without spending hours on it.

**Comment by tealou** [comment]
> In any case, it is really a shame to see that all kinds of new features are being introduced, instead of the current ones being improved. There are so many things that just don't work (right) lately. In addition, I would also like to see more advanced documentation rather than just standard solutions. A lot should be possible, but little is possible without spending hours on it.


Yep. Wouldn't we all. Welcome to the Crocoblock sunk costs fallacy ;)

**Comment by otomasz** [comment]
> Slick slider has become one of the worst sliders in developing world... Consider removing Slick sliders and converting to Swiper.js. Also, it's weird that you use swiper.js in woobuilder, but not in jetengine...
> 
> I would also like to see some more options like:
> 
> * Custom arrow container
> * Slides to show with decimal numbers
> * Free mode
> * scrollbar
> 
> Or just give the possibility to write some json for the advanced users.
> 
> In any case, it is really a shame to see that all kinds of new features are being introduced, instead of the current ones being improved. There are so many things that just don't work (right) lately. In addition, I would also like to see more advanced documentation rather than just standard solutions. A lot should be possible, but little is possible without spending hours on it.

right on point. widgets lack the basic rules of correct ux / ui

**Comment by beposss** [comment]
+1

**Comment by SolunaCG** [comment]
+1

**Comment by aalexandr21** [comment]
https://github.com/Crocoblock/suggestions/issues/6432#issuecomment-4046863631
